semeio/jobs/scripts/gendata_rft.py

```diff
@@ -173,14 +173,18 @@ def main_entry_point():
     options = arg_parser.parse_args()
     logger.setLevel(options.log_level)
 
-    well_names = [w_info[0] for w_info in options.well_and_time_file]
-
-    trajectories = {
-        wname: Trajectory.load_from_file(
-            filepath=os.path.join(options.trajectory_path, wname + ".txt")
-        )
-        for wname in well_names
-    }
+    context_errors = []
+    trajectories = {}
+    for well_name in [w_info[0] for w_info in options.well_and_time_file]:
```

Comment:
I have to admit, injecting the list comprehension directly into the for loop expression didn't really look that good.. If you think this is better than having the additional line above, then I guess it's okay. Perhaps it was just that I didn't expect to see it like that..

Reply:
I am fine either way, did it like this because it was not used for anything else.

Reply:
Think it is more readable now, do you agree?